nix/sources.json

```diff
@@ -71,6 +71,12 @@
         "url": "https://github.com/NixOS/nixpkgs/archive/f1707d8875276cfa110139435a7e8998b4c2a4fd.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
+    "oracle-suite": {
+        "branch": "master",
+        "repo": "https://github.com/makerdao/oracle-suite.git",
```

Comment:
Please note, that until this repo is private, one needs to add a replace configuration in .gitconfig
```
[url "git@github.com:"]
	insteadOf = https://github.com/
```